### PR TITLE
Request additional Fashion facets

### PR DESCRIPTION
**Pull request by lbiedinger**

### CODE REVIEW

File: lang/en.js [hunk 1]
```diff
@@ -82,6 +82,18 @@ export default {
     "COUNTRY": {
       "name": "Providing country"
     },
+    "proxy_dc_format": {
+      "name": "Technique"
+    },
+    //"proxy_dc_format": {
```
Comment (rwd) [inline lang/en.js]: Remove this. We have decided this is out of scope for now.

File: store/collections/fashion.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { defaultFacetNames } from '../search';
 
-const fashionFacetNames = ['CREATOR'].concat(defaultFacetNames);
+const fashionFacetNames = ['CREATOR', 'proxy_dc_format', 'proxy_dcterms_medium', 'proxy_dc_type'].concat(defaultFacetNames);
```
Comment (rwd) [inline store/collections/fashion.js]: Shouldn't these new facets all be `.en`? They are on Portal.rb

Comment (lbiedinger) [reply store/collections/fashion.js]: yes, this is causing some issues with the labelling I'll need to look into though.

Comment (lbiedinger) [reply store/collections/fashion.js]: I've  pushed changes to escape the translation keys. This is only applied to the more facet dropdown elements. I've not applied the same convention of facet name -> translation key modifications on date facets or the standard dropdowns, as they won't contain these kind of keys for now.

An alternative approach to this may be additional nesting in the locale file, but this would move the translations to different depth depending on the actual facet name and seems inconsistent to me.
